### PR TITLE
Add missing images to imageset

### PR DESCRIPTION
**Pull request by lmm**
## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

**Review by tmjd — APPROVED** [review]
LGTM

File: maintenance/image-options/imageset.md [hunk 1]
```diff
@@ -95,12 +97,18 @@ spec:
...
     digest: "sha256:0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef"
+  - image: "calico/windows-upgrade"
+    digest: "sha256:0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef"
   - image: "tigera/operator"
     digest: "sha256:0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef"
   - image: "tigera/key-cert-provisioner"
     digest: "sha256:0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef"
 ```
 
+>**Note**: The calico/windows-upgrade image is only if your cluster is running on AKS and has Windows nodes.
```
Comment (tmjd) [inline maintenance/image-options/imageset.md:108]: Should we drop this note now?

File: maintenance/image-options/imageset.md [hunk 2]
```diff
@@ -167,12 +175,7 @@ EOF
 for x in "${imagelist[@]}"; do
   for y in ${images[*]}; do
     if [[ $x =~ $y: ]]; then
-      echo "Pulling $x"
-      docker pull $x -q > /dev/null
-      {% raw %}digests=$(docker inspect $x -f '{{range .RepoDigests}}{{printf "%s\n" .}}{{end}}'){% endraw %}
-      name=$(echo $x | sed -e 's|docker.io/||' -e 's|:.*$||')
-      imgdigest=$(echo -e $digests | grep $name)
-      digest=$(echo -e $imgdigest | sed -e 's|^.*@||')
+      digest=$(docker run --rm gcr.io/go-containerregistry/crane digest ${x})
```
Comment (tmjd) [inline maintenance/image-options/imageset.md:178]: Is this a publicly available image?
I think we should include an image version. Should we copy the image to some place we control.

Comment (lmm) [reply maintenance/image-options/imageset.md]: Oops I meant to have the tag in there, will add it. Yes this image is publicly available on gcr: https://github.com/google/go-containerregistry/releases/tag/v0.7.0

**Comment by caseydavenport** [comment]
@lmm I believe this PR needs a rebase

**Review by caseydavenport — APPROVED** [review]

**Pull request merged**